src/config/config.ts

```diff
@@ -40,4 +41,23 @@ export class RaftConfig {
         return '';
     }
   }
+
+  static getTokenTicker(address: string): Token | null {
+    switch (address.toLowerCase()) {
+      case ZeroAddress:
+      case '':
```

Comment:
Do we want to return 'ETH' for empty address? It might lead to unexpected behaviour.

Reply:
Zero address, null or something else?

Reply:
I would say null.